chat/src/main/kotlin/net/thechance/chat/api/controller/ChatController.kt

```diff
@@ -83,6 +82,25 @@ class ChatController(
         return ResponseEntity.ok(message.toResponse(senderId))
     }
 
+    @PostMapping("/audio")
+    fun sendMessageAudio(
+        @ModelAttribute request: MessageAudioRequest,
+        principal: Principal
+    ): ResponseEntity<MessageResponse> {
+        val senderId = UUID.fromString(principal.name)
```

Comment:
```suggestion
    @PostMapping("/audio")
    fun sendMessageAudio(
        @ModelAttribute request: MessageAudioRequest,
        @AuthenticationPrincipal senderId: UUID,
    ): ResponseEntity<MessageResponse> {
```
I suggest to use AuthenticationPrincipal here because it's not messageMapping